treepoem/data.py

```diff
@@ -3,106 +3,106 @@
 
 # All supported barcode types, extracted from barcode.ps
 barcode_types = {
-    'auspost',
-    'azteccode',
-    'azteccodecompact',
-    'aztecrune',
-    'bc412',
-    'channelcode',
-    'codablockf',
-    'code11',
-    'code128',
-    'code16k',
-    'code2of5',
-    'code32',
-    'code39',
-    'code39ext',
-    'code49',
-    'code93',
-    'code93ext',
-    'codeone',
-    'coop2of5',
-    'daft',
-    'databarexpanded',
-    'databarexpandedcomposite',
-    'databarexpandedstacked',
-    'databarexpandedstackedcomposite',
-    'databarlimited',
-    'databarlimitedcomposite',
-    'databaromni',
-    'databaromnicomposite',
-    'databarstacked',
-    'databarstackedcomposite',
-    'databarstackedomni',
-    'databarstackedomnicomposite',
-    'databartruncated',
-    'databartruncatedcomposite',
-    'datalogic2of5',
-    'datamatrix',
-    'datamatrixrectangular',
-    'dotcode',
-    'ean13',
-    'ean13composite',
-    'ean14',
-    'ean2',
-    'ean5',
-    'ean8',
-    'ean8composite',
-    'flattermarken',
-    'gs1-128',
-    'gs1-128composite',
-    'gs1-cc',
-    'gs1datamatrix',
-    'gs1datamatrixrectangular',
-    'gs1northamericancoupon',
-    'gs1qrcode',
-    'hanxin',
-    'hibcazteccode',
-    'hibccodablockf',
-    'hibccode128',
-    'hibccode39',
-    'hibcdatamatrix',
-    'hibcdatamatrixrectangular',
-    'hibcmicropdf417',
-    'hibcpdf417',
-    'hibcqrcode',
-    'iata2of5',
-    'identcode',
-    'industrial2of5',
-    'interleaved2of5',
-    'isbn',
-    'ismn',
-    'issn',
-    'itf14',
-    'japanpost',
-    'kix',
-    'leitcode',
-    'matrix2of5',
-    'maxicode',
-    'micropdf417',
-    'microqrcode',
-    'msi',
-    'onecode',
-    'pdf417',
-    'pdf417compact',
-    'pharmacode',
-    'pharmacode2',
-    'planet',
-    'plessey',
-    'posicode',
-    'postnet',
-    'pzn',
-    'qrcode',
-    'rationalizedCodabar',
-    'raw',
-    'royalmail',
-    'sscc18',
-    'symbol',
-    'telepen',
-    'telepennumeric',
-    'ultracode',
-    'upca',
-    'upcacomposite',
-    'upce',
-    'upcecomposite',
+    'auspost': 'AusPost 4 State Customer Code',
```

Comment:
To avoid having to change the type again if we ever extract more data from the postscript file, e.g. the examples, I think it would be nice to make the values a custom class with attributes e.g.

```
class BarcodeType:
    def __init__(self, type_code, description):
        self.type_code = type_code
        self.description = description

barcode_types = {
    'auspost': BarcodeType('auspost', 'AusPost 4 State Customer Code'),
```

Then future changes just extend `BarcodeType`, which *is* backwards compatible :)

I know it's a bit more work, but would you be able to do this?